app/portainer/services/localStorage.js

```diff
@@ -59,6 +59,16 @@ angular.module('portainer.app')
     storeDataTableSettings: function(key, data) {
       localStorageService.set('datatable_settings_' + key, data);
     },
+    storeSwarmVisualizerSettings: function(key, data) {
+      localStorageService.set('swarmvisualizer_' + key, data);
+    },
+    getSwarmVisualizerSettings: function(key, defaultVal) {
```

Comment:
I believe that the `LocalStorage` service purpose should be to store/retrieve values from the local storage, not to manage default values.

To stay consistent with how this is done currently, could you relocate the default variable handling into the `initView` function of the `SwarmVisualizerController`?